RecFCCeeCalorimeter/CMakeLists.txt

```diff
@@ -45,6 +45,12 @@ add_test(NAME FCCeeLAr_simulateForReco
 )
 set_test_env(FCCeeLAr_simulateForReco)
 
+add_test(NAME FCCeeLAr_simRecoAllegroV2
```

Comment:
Shall we set the name to v3 instead of v2 since it actually tests `ALLEGRO_o1_v03` ?

Reply:
Changed by @AlexisMlz in latest commit